.gitignore

```diff
@@ -109,3 +109,6 @@ chart/k8gb/charts/*.tgz
 
 # Ignore testing kubeconfigs
 kubeconfig*
+
+# Ignore changes from github_changelog_generator action
+changes
```

Comment:
How do we update `CHANGELOG.md` in this scenario?

Reply:
same way as before, post release commit (post release github_changelog_generator on maintainers machine and open PR). 
During GH run only temporary diff is generated into gitingored file.

Reply:
But before local github_changelog_generator diff was the input for release. Now it will be required to run it locally post release and commit? It might lead to deviations

Reply:
If I look at last v0.7.6 release, offline changelog was commited at 18:00, and release was done on 15:06. I see no deviation.

Reply:
yeas, because offline changelog is the source for release text

Reply:
Is it worth to sacrafice automation in order to have "offline" change log?
you have two tags, and tool to work with those tags. ATM we run tool get output and paste it into "Release".
I propose to run tool as part of release draft pipeline to format a release. Then run tool with same input to commit post release offline changelog. Sorry I only see an improvement here.

Additionally draft can be changed the same way from "offline" changelog.

Reply:
I am not talking about sticking to current scenario, more thinking of avoiding pitfalls with the new one. Can we can write to CHANGELOG.md in post-release pipeline and commit it automatically?

Reply:
This option can be explored for sure.

Reply:
@ytsarev ammended

Reply:
@k0da very cool, thanks!